client/src/assets/scss/trails.scss

```diff
@@ -1,7 +1,13 @@
-body, html {
+body,
+html {
   height: 100%;
 }
 
+.button-group {
+  border-radius: 20px;
+  background: #f17b30;
```

Comment:
If this is a class being used in multiple places you can just define it once in `main.scss` rather than repeated multiple times